src/imagery/i.eodag/i.eodag.html

```diff
@@ -282,6 +282,19 @@ <h3>EODAG configuration</h3>
           password: password
 </pre></div>
 
+<h3>Creodias</h3>
+Using the provider Creodias is slightly different, since it requires TOTP for authentication.
+To register to credoias create an account
+<a href="https://portal.creodias.eu/register.php">here</a>,
+then use your username, password in eodag configuration file.
+You will also need totp, a temporary 6-digits OTP
+(One Time Password, see Creodias documentation) to be able to authenticate
+and download scenes, which shall not be set through the eodag configuration file,
+but instead a prompt will show asking for OTP in case <em>i.eodag</em> attempts to
+download a scene from creodias. To discard these scenes the prompt will
+ask you to input <b>"-"</b> instead of the OTP. Note that interactive prompt
+does not work in the graphical user interface.
```

Comment:
```suggestion
<h4>Creodias</h4>
The Creodias provider requires a temporary one time password (TOTP) 
for authentication. To register to Creodias, users should create an account
<a href="https://portal.creodias.eu/register.php">here</a>,
then use their username and password in the eodag configuration file.
Users will also need totp, a temporary 6-digits OTP
(One Time Password, see Creodias documentation) to be able to authenticate
and download scenes. This topt shall not be set through the eodag configuration file,
but instead a prompt will show asking for it in case <em>i.eodag</em> attempts to
download a scene from creodias. To discard these scenes users will be prompted 
to input <b>"-"</b> instead of the OTP. Note that interactive prompt
does not work in the graphical user interface.
```

I checked the text and try fixing it but I'm not sure I understand this correctly; users need two TOTP for creodias?? One in the config file and then another one for download?? Or they are the same thing?

Reply:
No they are the same thing, what I am trying to convey is that users shouldn't set the totp in the config file, since it changes every 30 seconds or so anyways...
So they need to add it on the spot to be able to authenticate within the 30 seconds time frame.

Reply:
updated now, check if that makes sense

Reply:
> updated now, check if that makes sense

It is more clear now thanks!